test/tables/sortable-table.test.js

```diff
@@ -202,11 +202,26 @@ test('initialColumn determines inital sortPath and sortFunc', () => {
   expect(mySort).toHaveBeenCalled()
 })
 
+test('initialColumn can be default sorted descending', () => {
```

Comment:
Also add a test case for sorting with valueGetter to make sure the bug is fixed!

Reply:
Done.